transmart-rest-api/grails-app/services/org/transmartproject/rest/dataExport/RestDataExportService.groovy

```diff
@@ -0,0 +1,77 @@
+package org.transmartproject.rest.dataExport
+
+import grails.transaction.Transactional
+import org.transmartproject.core.exceptions.InvalidRequestException
+import org.transmartproject.core.exceptions.LegacyStudyException
+import org.transmartproject.core.users.ProtectedOperation
+import org.transmartproject.core.users.User
+import org.transmartproject.db.multidimquery.query.Constraint
+import org.transmartproject.db.multidimquery.query.OrConstraint
+import org.transmartproject.db.multidimquery.query.PatientSetConstraint
+import org.transmartproject.rest.MultidimensionalDataService
+import org.transmartproject.rest.MultidimensionalDataService.Format as Format
+import sun.reflect.generics.reflectiveObjects.NotImplementedException
+
+import java.util.zip.ZipOutputStream
+
+
+@Transactional
+class RestDataExportService {
```

Comment:
it would be nice to resolve it. transmartApp will be out of the picture at some point.